lib/email/receiver.rb

```diff
@@ -558,7 +558,10 @@ def reply_above_line_regex
 
     def trim_discourse_markers(reply)
       reply = reply.split(previous_replies_regex)[0]
```

Comment:
The `split` returns an empty array only when `reply` is empty as well. I think we should bail out early in that case.

```ruby
def trim_discourse_markers(reply)
  return "" if reply.blank?
  reply = reply.split(previous_replies_regex)[0]
  reply = reply.split(reply_above_line_regex)[0]
end
```

Reply:
change made